addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -106,10 +236,209 @@
+
+    <channel-type id="images" advanced="true">
+        <item-type>String</item-type>
+        <label>Images</label>
+        <description>The available images as a json array</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="locationName" advanced="true">
+        <item-type>String</item-type>
+        <label>Location Name</label>
+        <description>The location of where the current media was taken</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="locationLatitude" advanced="true">
+        <item-type>Number</item-type>
+        <label>Location Latitude</label>
+        <description>The location of where the current media was taken</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="locationLongitude" advanced="true">
+        <item-type>Number</item-type>
+        <label>Location Longitude</label>
+        <description>The location of where the current media was taken</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="releaseDate" advanced="true">
+        <item-type>Number</item-type>
+        <label>Release Date</label>
+        <description>The release date of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="seasonNumber" advanced="true">
+        <item-type>Number</item-type>
+        <label>Season Number</label>
+        <description>The season number of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="seriesTitle" advanced="true">
+        <item-type>String</item-type>
+        <label>Series Title</label>
+        <description>The series title of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="studio" advanced="true">
+        <item-type>String</item-type>
+        <label>Studio</label>
+        <description>The studio of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="subtitle" advanced="true">
+        <item-type>String</item-type>
+        <label>Subtitle</label>
+        <description>The subtitle of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="title" advanced="true">
+        <item-type>String</item-type>
+        <label>Title</label>
+        <description>The title of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="trackNumber" advanced="true">
+        <item-type>Number</item-type>
+        <label>Track Number</label>
+        <description>The track number of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="width" advanced="true">
```

Comment:
As mentioned above, Google added it and documented it, so I'm not sure it makes sense for us to exclude it.